packages/lambda-tiler/src/util/config.loader.ts

```diff
@@ -14,9 +14,9 @@ export class ConfigLoader {
   static async getDefaultConfig(): Promise<BasemapsConfigProvider> {
     const config = getDefaultConfig();
     if (config.assets == null) {
-      const cb = await config.ConfigBundle.get(config.ConfigBundle.id('latest'));
-      if (cb == null) throw new LambdaHttpResponse(400, 'Unable to get lastest config bundle for asset.');
-      config.assets = cb.assets;
+      // const cb = await config.ConfigBundle.get(config.ConfigBundle.id('latest'));
```

Comment:
Why do we comment out this? config assets is broken?

Reply:
running a server without this commented out, causes the server to crash unless you specify a assets and you dont need the assets for most actions of the server.

but this shouldnt be left commented out.